tpm/info.go

```diff
@@ -14,13 +15,16 @@ import (
 // TPM version, interface, manufacturer, vendor info and
 // firmware version.
 type Info struct {
-	Version         Version         `json:"version"`
-	Interface       Interface       `json:"interface"`
-	Manufacturer    Manufacturer    `json:"manufacturer"`
-	VendorInfo      string          `json:"vendorInfo,omitempty"`
-	FirmwareVersion FirmwareVersion `json:"firmwareVersion,omitempty"`
+	Version             Version         `json:"version"`
+	Interface           Interface       `json:"interface"`
+	Manufacturer        Manufacturer    `json:"manufacturer"`
+	VendorInfo          string          `json:"vendorInfo,omitempty"`
+	FirmwareVersion     FirmwareVersion `json:"firmwareVersion,omitempty"`
+	SupportedAlgorithms []Algorithm     `json:"supportedAlgorithms"`
```

Comment:
Not an illogical place to put it, but I think exporting a new method on `TPM` will be better in the long term. It can be marked experimental, so that we can change the name, remove it later, etc. if we decide that's the best way forward.